localization/english/wc_regiment_l_english.yml

```diff
@@ -0,0 +1,12 @@
+﻿l_english:
+#MaA types
+ caster:2 "[caster|E]"
+ light_aerial:2 "[light_aerial|E]"
+ massive_aerial:2 "[massive_aerial|E]"
+ massive_infantry:2 "[massive_infantry|E]"
```

Comment:
`massive_infantry` doesn't exist as a game concept thus there's an empty line for the giants.
Also, `giant_unit_flavor` isn't localized.
The same problem with other new units.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/102015397-d260ec00-3d74-11eb-86ba-aa69c5079c5a.png)

</details>